apps/web-giddh/src/assets/locale/sidebar-menu/en.json

```diff
@@ -92,7 +92,7 @@
                 },
                 {
                     "label": "Invoice Settings",
-                    "link": "/pages/invoice/preview/settings/sales",
+                    "link": "/pages/vouchers/preview/sales/settings",
```

Comment:
define separately for both voucher versions

Reply:
Done